### ISSUE TITLE
[STCATP] Support ticket changes #1582

### ISSUE THREAD

**Issue by nupoorkhandelwal** [avniproject/stcatp#44]
- [x] Add NA as location from block till lowest address levels. Also add it to program manager's catchments **Remove this change**

- [x] Hide location form element group when NA is selected as the block in the event reporting - **Remove this change**

- [x] Create new subject type District level Event reporting 

- [x] REACH report- Add district filter

- [x] For Monthly Planner - create a user type location and add it to the user's catchment. User should be able to register Monthly planner on this level and schedule visits for themselves

**Comment by nupoorkhandelwal** [comment]
@garimadosar5 - **First two changes are done on UAT.**

**Comment by garimadosar5** [comment]
QA Notes - 
- After block is selected NA, Village and other locations are showing as NA and reg is not going forward.
![Screenshot_2022-01-28-22-38-58-98.jpeg](https://images.zenhubusercontent.com/5d0a151d5af8b352a1b5784b/e843a1bd-26a1-43d3-aa10-3706a0bf94e0)

**Comment by nupoorkhandelwal** [comment]
On UAT, for monthly planner registration location won't work write now. The solution is still not released. It will work once release is done.

**Comment by garimadosar5** [comment]
@nupoorkhandelwal Then this card is not done. Moving this on Hold.
